circuit-lib/circuit-lib.js/tsconfig.json

```
@@ -16,7 +16,7 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true
   },
-  "include": ["src", "__tests__", "tests"],
+  "include": ["src", "__tests__", "tests", "../circuit-lib.circom/tests/elgamal-babyjubjub/circuits.test.ts", "index.ts"],
```

```suggestion
  "include": ["src"],
```
circuit-lib.circom is it's own package why include it here?

---

I removed this before, now it is updated. This prevented `system-programs` tests from running -> fixed